frameworks/nextjs/README.md

```diff
@@ -1,15 +1,20 @@
-# NextJS
+# Next.js
 
-NextJS with SSR and `getStaticProps`.
+This project uses Prisma Client in [Next.js](https://nextjs.org/) site with [SSR](https://nextjs.org/docs/basic-features/pages#server-side-rendering) and [`getStaticProps`](https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation).
 
-## How to run this locally
+## About this project
 
 ### Environment variables
 
-The environment variable `FRAMEWORK_NEXTJS_PG_URL` should point to a postgres database.
-In CI, it uses our internal e2e test database using `framework-nextjs` as database URL.
-Please check our internal 1Password E2E vault for a ready-to-use environment variable or  
-set up your own database and set the environment variable accordingly.
+- The environment variable `FRAMEWORK_NEXTJS_PG_URL` should point to a PostgreSQL database.
+- In CI, it uses our internal E2E test database host using `framework-nextjs` as database.
+- Please check our internal 1Password E2E vault for a ready-to-use environment variable value or set up your own database and set the environment variable accordingly.
+
+### Run project
+
+```shell script
+sh test.sh
```

Comment:
🤔 I guess it depends what you mean by "run project"

Reply:
run.sh and test.sh were obviously mixed up here.

not super happy about the naming there, but not mentioning the second script that is there was wrong.
suggestions for better naming?

Reply:
It's universal so we can probably ignore it.